docs/srs.md

```diff
@@ -0,0 +1,127 @@
+# SOFTWARE REQUIREMENTS SPECIFICATION
+
+## Prepared by
+
+### Mykyta Kyt | Alex Zhytnyk
+
+1. Introduction
+
+  - 1.1 Purpose
+  - 1.2 References
+
+2. Overall Description
+
+  - 2.1 Product perspective
+  - 2.2 Product features
+  - 2.3 Operating environment
+  - 2.4 Design and implementation constraints
+
+3. System features
+
+  - 3.1 Description and priority
+
+4. External interface requirements
+
+  - 4.1 User Interfaces
+  - 4.2 Hardware Interfaces
+
+5. Non functional requirements
+
+  - 5.1 Software quality attributes
+  - 5.2 Security requirements
+
+## 1\. Introduction
+
+### 1.1 Purpose
+
+The purpose of this document is to describe the equipment and employee's accounting system in the company. Of itself, it will represent a database to which you can connect through the web client. The database will have a lot of features such as auto back-up, export to .csv or .xls files, create reports, create new fields, delete and update them. It will be possible to generate inventory labels, which means qr codes. The database will contain all the information about employees and equipment. This product will be able to organize everything and make life easier.
+
+### 1.2 References
+
+1. <https://www.wikiwand.com/en/Flask_(web_framework)> : Flask is a python based lightweight web application framework.
+2. <https://www.postgresql.org/> : PostgreSQL is a powerful, open source object-relational database system.
+3. <https://pypi.org/project/qrcode/>: Python's library for generating unique qr code for each device.
+4. <https://cutt.ly/zf7MkUX> : This article is an explanation of how to connect PostgreSQL with Flask.
+5. <https://hydra.cc/docs/intro/> : Hydra is an open-source Python framework that simplifies the development of research and other complex applications.
+
+## 2\. Overall Description
+
+### 2.1 Product perspective
+
+An accounting database system stores the following information.
+
+- **Equipment details:** It includes information about the equipment, such as laptops, monitors, mice, keyboards, headphones, which are issued to employees, as well as information about the equipment that is needed in other cities.
+- **Employee description:** It includes the employee's id, first name, last name, position, mail, phone number, birthday, city, project.
+- **Description of interaction:** It includes information which technique is given to each employee or information that the employee works on his technique. It also includes information about the workplace.
+
+### 2.2 Product features
+
+This item presents the DFD-diagram, which shows the general cooperation in the system. They are presented in pictures 1.1-1.2 [![Picture 1.1 - DFD diagram A-0 level.](https://i.ibb.co/Jjn6rqM/Untitled-Document.png)](https://ibb.co/r2k8531)
```

Comment:
Store images in the repo

Reply:
How to add it there?

Reply:
Create a folder `images`, put all needed images there and commit.
In MD use relative path
```
![Image Name](images/image.png)
```